src/serializer/builtins.ts

```diff
@@ -74,6 +75,10 @@ export interface BuiltinTypes {
     'extended_asset?'?: ExtendedAsset
     'extended_asset[]': ExtendedAsset[]
     'extended_asset[]?'?: ExtendedAsset[]
+    blob: Blob
```

Comment:
Blobs are not builtins

Reply:
Removed in https://github.com/wharfkit/antelope/pull/71/commits/a0684a66c972c46d3736d19cf906f423d714a1fe